### PR TITLE
APP-1301 - Removed Click rings 

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- focus rings have been removed

Note: whoever wants to review, please reach out to me first

Task: [APP-1301](https://aragonassociation.atlassian.net/browse/APP-1301)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

**Comment by Fabricevladimir** [comment]
Hey @sepehr2github, when I run Storybook, the outlines are removed on click, but focusing with the keyboard is generating a black outline instead of the `primary-500` ring. :/

**Comment by sepehr2github** [comment]
> Hey @sepehr2github, when I run Storybook, the outlines are removed on click, but focusing with the keyboard is generating a black outline instead of the `primary-500` ring. :/

**Comment by sepehr2github** [comment]
> Hey @sepehr2github, when I run Storybook, the outlines are removed on click, but focusing with the keyboard is generating a black outline instead of the `primary-500` ring. :/

which input??? I could find sth similar!

**Comment by sepehr2github** [comment]
Just a head up, Input rings got the approval from Marco

**Review by Fabricevladimir — APPROVED** [review]

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Hey @sepehr2github, thanks a lot for the work. Very nearly looks like what we want 🎉 

Unfortunately, there's still a little issue. If you zoom in onto a button, for example, you'll see two things: 
a) the ring do not quite have the right color (particularly noticeable in firefox)
b) there is a few pixels of white space around the blue rings (both firefox and chrome)

image

It turns out that this is because of this part of the TW config: 
https://github.com/aragon/apps/blob/2cb7b122f5e1906961e6c56d7701c434e1aa5aa2/packages/ui-components/tailwind.config.js#L169-L171

If you remove these lines, the issue shown in the picture goes away and the rings are shown with proper width and color.

File: packages/ui-components/src/components/input/dateInput.tsx [hunk 1]
```diff
@@ -59,6 +59,8 @@ const StyledInput = styled.input.attrs(() => {
     margin-top: 4px;
     margin-bottom: 4px;
   }
+
+  outline: 0;
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/input/dateInput.tsx:63]: Minor detail: Can we use `focus-visible:outline-none`?

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Hey Sepehr, thanks for the changes. Seems to work 🤞🏼

While I was browsing the app I noticed a clickable component (that is not part of the UI component library) that hasn't been updated: https://github.com/aragon/apps/blob/5f5d237e46e4ec666c675e620dd23cbd5a886ef5/packages/web-app/src/components/navLink/index.tsx#L84-L94

Sorry I didn't catch that earlier 🙏🏼

File: packages/ui-components/src/components/input/dateInput.tsx (hunk 1 above)
Comment (sepehr2github) [reply packages/ui-components/src/components/input/dateInput.tsx:63]: Tested this solution, It's not working for some reason!

**Review by WissenIstNacht — APPROVED** [review]
Let's gooo 👍🏼

**Pull request merged**